keyboards/ut47/LED_controls.ino

```diff
@@ -0,0 +1,420 @@
+#include <FastGPIO.h>
```

Comment:
Yes, this file is required because this keyboard uses 2 pro micros: one that is flashed with QMK and controls the keyboard, and a second that is used to control the per-key LEDs. While not required for the keyboard to work, its required for a full LED build.

Reply:
I think that it may be better to host the actual hex file, so people can flash it without having to set up more stuff to flash it.  (as it appears that you need Arduino Studio and the FastGPIO library installed. 

@jackhumbert in this case, would hosting the secondary HEX file in github here be fine?

Reply:
Good call on the compiled binary. I'll need to investigate something unrelated to QMK before I can do that. The .hex file is 19K or 28K depending on whether it includes the bootloader, and knowing that might be helpful for deciding if I can leave it here. I'd prefer to have that file closer to the keymap source as much as possible for ease of use and installation; navigating and using QMK is hard enough. I'd like to reduce as much friction as possible.